setup.py

```diff
@@ -68,6 +74,6 @@ def run(self):  # noqa: D102
     packages=find_packages(exclude=["tests"]),
     package_data={"pyramid_openapi3": ["static/*.*"], "": ["LICENSE"]},
     install_requires=["openapi-core", "openapi-spec-validator", "pyramid"],
-    extras_require={':python_version<"3.7"': ["importlib-resources"]},
+    extras_require={"testing": testing_extras, ':python_version<"3.7"': ["importlib-resources"]},
```

Comment:
We should then have `.[testing]` (or sth. similar) in Pipfile, under `development` section, so that we don't have two lists of development/testing dependencies.

Reply:
I don't know pipfile, but I added stuff in 9c18f73 for review.

Reply:
Updated in 1df7ebd per https://github.com/Pylons/pyramid_openapi3/pull/30#issuecomment-518036090